test/chef_keyring_tests.erl

```diff
@@ -123,11 +130,45 @@ load_file_test_() ->
      ]
     }.
 
+load_chef_secrets_test_() ->
```

Comment:
[nit] This test isn't really about chef-secrets - it's more about testing the callback functionality. Maybe s/chef_secrets/secrets_callback_mod or similar?